overrides/scripts/villager_trades_removed.zs

```diff
@@ -0,0 +1,128 @@
+
+//villagerTrades.removeBasicTrade(<profession:minecraft:farmer>, 1, <item:minecraft:bread>); // used mostly for modded recipes and not vanilla villager trades
+
+
+//remove trades to get emeralds from Farmers
+villagerTrades.removeAllTrades(<profession:minecraft:farmer>, 1);
+villagerTrades.removeTrade(<profession:minecraft:farmer>, 1, <item:farmersdelight:tomato>, <item:minecraft:emerald>);
+villagerTrades.removeTrade(<profession:minecraft:farmer>, 1, <item:farmersdelight:onion>, <item:minecraft:emerald>);
+
+
+//remove trades to get emeralds from Beekeepers
+villagerTrades.removeTrade(<profession:friendsandfoes:beekeeper>, 1, <item:minecraft:sunflower>, <item:minecraft:emerald>);
+villagerTrades.removeTrade(<profession:friendsandfoes:beekeeper>, 1, <item:minecraft:dandelion>, <item:minecraft:emerald>);
+villagerTrades.removeTrade(<profession:friendsandfoes:beekeeper>, 1, <item:friendsandfoes:buttercup>, <item:minecraft:emerald>);
+
+//add trade to get Busy Bee enchant from Beekeepers
+villagerTrades.addTrade(<profession:friendsandfoes:beekeeper>, 1, (entity, random) => 
+{
+var priceChooser = random.nextInt(3)+1;
+if (priceChooser == 3)
+{
+    return new MerchantOffer(<item:minecraft:emerald> * (random.nextInt(10)+35), <item:minecraft:book>, <item:minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 3, id: "dungeons_gear:busy_bee"}]}), 16, 0, 5) as MerchantOffer;
+}
+else if (priceChooser == 2)
+{
+    return new MerchantOffer(<item:minecraft:emerald> * (random.nextInt(10)+25), <item:minecraft:book>, <item:minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 2, id: "dungeons_gear:busy_bee"}]}), 16, 0, 5) as MerchantOffer;
+}
+else
+{
+    return new MerchantOffer(<item:minecraft:emerald> * (random.nextInt(10)+15), <item:minecraft:book>, <item:minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 1, id: "dungeons_gear:busy_bee"}]}), 16, 0, 5) as MerchantOffer;
+}
+});
+
+val villagerPro = [
+<profession:blue_skies:alchemist>,
+<profession:blue_skies:nightwatcher>,
+<profession:blue_skies:shoveler>,
+<profession:blue_skies:stargazer>,
+<profession:blue_skies:summoner>,
+<profession:blue_skies:wrangler>,
+<profession:domesticationinnovation:animal_tamer>,
+<profession:friendsandfoes:beekeeper>,
+<profession:minecraft:armorer>,
+<profession:minecraft:butcher>,
+<profession:minecraft:cartographer>,
+<profession:minecraft:cleric>,
+<profession:minecraft:farmer>,
+<profession:minecraft:fisherman>,
+<profession:minecraft:fletcher>,
+<profession:minecraft:leatherworker>,
+<profession:minecraft:librarian>,
+<profession:minecraft:mason>,
+<profession:minecraft:nitwit>,
+<profession:minecraft:none>,
+<profession:minecraft:shepherd>,
+<profession:minecraft:toolsmith>,
+<profession:minecraft:weaponsmith>
+];
+
+for prof in villagerPro 
+{
+    villagerTrades.removeTradesSelling(prof, 1, <item:minecraft:emerald>);
+}
+
+val toBuy = [<item:farmersdelight:barbecue_stick>,
```

Comment:
things inside brackets should start on a new line, indented to show it is nested inside the brackets

so 
```js
val toBuy = [
  <item>
]
```